src/main/java/models/Game.java

```diff
@@ -87,6 +87,20 @@ else if (dealerScore != userScore)
 			userWon = true;
 	}
 
+    // A player choosing to stay is essentially a signal
+    // for the dealer's turn to start
+    public void stay(){
+        dealCardToDealer(); // Call twice at turn start
+        dealCardToDealer();
+        // Do we need to turn one of these cards face down? Or will that
```

Comment:
All right. I know the card class has two boolean functions that will set whether it should be face up or not. Maybe you can use those?